src/background.js

```diff
@@ -86,6 +86,18 @@ function sendMetrics(event, context = {}, sync = reduxSync) {
   timeouts[event] = setTimeout(later, 20000);
 }
 
+function fetchProfile(credentials) {
+  fxaFetchProfile(FXA_PROFILE_SERVER, credentials.access_token).then((profile) => {
```

Comment:
this request may fail , for example due to an expired token. 

If this request fails we should just "log out" and user can resignin in that case

Reply:
basically do we need to catch the error case here or it doesn't matter?

Reply:
I believe logout will occur because kinto-sync will fail using expired credential but just to be sure I pushed a catch sending `reconnect` message.